assets/prometheus-adapter/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
```

Comment:
add this annotation to get a serving TLS cert:

```
metadata:
  annotations:
    service.alpha.openshift.io/serving-cert-secret-name: prometheus-adapter-tls
```

then mount the `prometheus-adapter-tls` secret into the deployment and use the cert within that. The keys are called `tls.crt` and `tls.key`.